HIRS_Utils/src/main/java/hirs/data/persist/ComponentInfo.java

```diff
@@ -216,9 +220,8 @@ public static boolean isComplete(final String componentManufacturer,
                                      final String componentModel,
                                      final String componentSerial,
                                      final String componentRevision) {
-        return !(
-                StringUtils.isEmpty(componentManufacturer)  || StringUtils.isEmpty(componentModel)
-        );
+        return !(StringUtils.isEmpty(componentManufacturer)
+                || StringUtils.isEmpty(componentModel));
```

Comment:
This is more of a stylistic issues, but I know I reason about this better when we do a simplification of these lines to simply `return !StringUtils.isEmpty(componentManufacturer) && !StringUtils.isEmpty(componentModel);` This keeps the statement to "componentManufacturer is not empty AND componentModel is not empty". It will work either way, but one seems easier to interpret at a passing glance.

Reply:
Will look over again.  Looking at this again, it looks odd to me as well.